README.md

```diff
@@ -0,0 +1,330 @@
+```
+
+#### ShrinkWrap Resolver Strategies
+
+In ShrinkWrap Resolver, strategies allow you to modify the dependency graph and change the default behavior for resolving transitive dependencies.
+
+By default, Maven and ShrinkWrap Resolver do not resolve dependencies in provided and test scope, and skip optional dependencies.
+
+#### Available Strategies:
+
+- **AcceptAllStrategy (Equals TransitiveStrategy):**
+  Accepts all dependencies of artifacts.
+
+- **AcceptScopesStrategy:**
+  Accepts only dependencies with defined scope types.
+
+- **CombinedStrategy:**
+  Combines multiple strategies with a logical AND operation.
+
+- **NonTransitiveStrategy:**
+  Rejects all transitive dependencies; only directly specified dependencies are accepted.
+
+- **RejectDependenciesStrategy:**
+  Rejects dependencies defined by G:A. By default, it is transitive, meaning all dependencies originating at G:A are removed. To change this behavior to reject defined dependencies but keep their descendants, use `RejectDependenciesStrategy(false, "G:A")`.
+
+- **TransitiveStrategy (Equals AcceptAllStrategy):**
+  Accepts all dependencies of artifacts.
+
+
+#### Control Sources of Resolution
+
+ShrinkWrap Resolvers allows you to specify artifact resolution sources. By default, it uses the classpath (Maven Reactor) and Maven Central repository.
+
+```java
+Maven.configureResolver().withClassPathResolution(false).resolve("G:A:V").withTransitivity().asFile();
+Maven.configureResolver().withMavenCentralRepo(false).resolve("G:A:V").withTransitivity().asFile();
+Maven.configureResolver().workOffline().resolve("G:A:V").withTransitivity().asFile();
+Maven.configureResolver().useLegacyLocalRepo(true).resolve("G:A:V").withTransitivity().asFile();
+```
+
+#### Specify settings.xml
+```java
+Maven.configureResolver().fromFile("/path/to/settings.xml")
+  .resolve("G:A:V").withTransitivity().asFile();
+
+Maven.configureResolver().fromClassloaderResource("path/to/settings.xml")
+  .resolve("G:A:V").withTransitivity().asFile();
+```
+
+<sub>**Warning:** ShrinkWrap Resolvers doesn't consume settings.xml specified on the command line or in the IDE. It reads settings.xml files at standard locations: `~/.m2/settings.xml` and `$M2_HOME/conf/settings.xml`, unless overridden in the API or via System property.</sub>
+
+#### Define Maven repositories manually
+
+```java
+Maven.configureResolver().withRemoteRepo("my-repository-id", "url://to/my/repository", "layout")
+  .resolve("G:A:V").withTransitivity().asFile();
+
+Maven.configureResolver().withRemoteRepo(MavenRemoteRepositories.createRemoteRepository("my-repository-id", "url://to/my/repository", "layout"))
+  .resolve("G:A:V").withTransitivity().asFile();
+```
+
+### Resolution of Artifacts from POM Files
+
+In Maven projects, you likely specify dependencies in your pom.xml file. ShrinkWrap Resolvers allows you to avoid redundancy by automatically loading metadata from there.
```

Comment:
These two sentences seem like they've been extracted from two different paragraphs. I don't really see a connection there. Perhaps instead of redundancy we should focus more on convenience? 

Or how do you understand them? @xjusko